<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddGameDialog</class>
 <widget class="QWidget" name="AddGameDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="addGame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>211</width>
     <height>335</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="gameName"/>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Producer</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="gameProducer"/>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Publisher</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="gamePublisher"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Category</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="gameCategory">
      <item>
       <property name="text">
        <string>Fantasy</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sci-Fi</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Drama</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Documentary</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Action</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Historycal</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Comedy</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Romance</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Introduction</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="gameIntroduction"/>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Release date</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="gameRelease">
      <property name="maximum">
       <number>2021</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="cancelAction">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>370</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
